dns/dns_updater.go

```diff
@@ -62,17 +75,19 @@ func (u *updater) String() string {
 func (u *updater) Start() error {
 	log.Info("Starting dns updater")
 
-	if u.elbLabelValue != "" && len(u.albNames) > 0 {
-		return fmt.Errorf("can't specify both elb label value (%s) and alb names (%v) - only one or the other may be"+
-			" specified", u.elbLabelValue, u.albNames)
-	}
+	if len(u.schemeToDNS) == 0 {
```

Comment:
I don't think you need this conditional, as everything in the block is a noop since elbLabelValue == "" and len(u.albNames) == 0.